src/components/Footer.css

```diff
@@ -135,8 +135,9 @@
   .footer-logo {
     z-index: 1;
     display: flex;
-    width: 130px; /* Not the cleanest fix sorry but it works as long as we don't resize the logo*/
+    width: 150px; /* Not the cleanest fix sorry but it works as long as we don't resize the logo*/
```

Comment:
This is acceptable as long as this is the constant logo size that we will be using, and looks good with all screen resolutions